generator/config/accumulator/accumulator.yaml

```diff
@@ -64,14 +64,29 @@ tests:
                     avgCopies:
                         $accumulator:
                             init:
-                                $code: 'function () { return { count: 0, sum: 0 } }'
+                                $code: |-
```

Comment:
Noted that this has no impact on `tests/Builder/Accumulator/Pipelines.php` apart from whitespace changes. Likewise for `query/where.yaml`.